client/src/ui/organisms/placement/index.scss

```diff
@@ -1,10 +1,16 @@
-.place {
+section.place {
   display: flex;
   flex-direction: column;
   margin: 1rem 0;
   max-width: 12rem;
   min-height: 20rem;
 
+  a:not(.button),
+  a {
+    color: var(--text-primary);
+    text-decoration: none;
+  }
```

Comment:
This change seems unrelated. What's the intention?

Are there also buttons inside this section?